src/js/components/Tabs/Tabs.js

```diff
@@ -39,6 +57,74 @@ const Tabs = forwardRef(
     delete rest.onActive;
     /* eslint-enable no-param-reassign */
 
+    const tabRefs = React.Children.map(children, () => React.createRef());
+
+    // check if tab is in view
+    function isVisible(element) {
```

Comment:
```suggestion
    const isVisible = (element) => {
```
Feel like this is more aligned with our conventions

Reply:
Agreed, I'll make this change

Reply:
Fixed